frontend/src/components/User/User.scss

```diff
@@ -1,12 +1,64 @@
 @import "../../styles/prelude.scss";
 
-.follow-button{
-    color: rgb(25, 136, 240);
-    background-color: rgb(40, 40, 40);
-    outline: none;
-    border: none;
-    padding: 5px 20px;
-    text-transform: uppercase;
-    border-radius: 2px;
-    cursor: pointer;
+.user-component {
+    .follow-button{
+        color: rgb(25, 136, 240);
+        background-color: rgb(40, 40, 40);
+        outline: none;
+        border: none;
+        padding: 5px 20px;
+        text-transform: uppercase;
+        border-radius: 2px;
+        cursor: pointer;
+    }
+    .all-posts {
+        box-sizing: border-box;
+        margin: 0 auto;
+        columns: 3;
+        column-gap: 7px;
+        max-width: 1440px;
+
+        .single-post .image-wrapper {
```

Comment:
Might just be the way I'm seeing it but check the indentation here.

Reply:
The indentation seems fine to me (and to VSCode....)